### PR TITLE
feat(rss): add RSS feed UI

### PR DESCRIPTION
**Pull request by Jasmine-liang**
Add RSS feed icon to user profile; 
Add RSS dialog;

### CODE REVIEW

**Comment by robertu7** [comment]
image

need a hover/focus effect here.

File: src/common/utils/resolvers/rssFeedGatewayUrls.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import fetch from 'isomorphic-unfetch'
```
Comment (robertu7) [inline src/common/utils/resolvers/rssFeedGatewayUrls.ts:1]: i think we can reuse `gatewayUrls`.

**Comment by tx0c** [comment]
rebase and squash ? into single commit as 26dc4dcc5 testing in `release/next`

File: src/components/Dialogs/RssFeedDialog/Content.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                </li>
+              )
+            })}
+          </ul>
+        </section>
+
+        <hr style={{ margin: '1rem 0' }} />
+
+        {/* hash */}
+        <section className="hash">
```
Comment (tx0c) [inline src/components/Dialogs/RssFeedDialog/Content.tsx:159]: the whole section should be conditionally displayed, aka. don't show it when there's no `ipnsKey`

```js
{ipnsKey && <section ...
}
```

**Pull request closed without merging**